docs/source/json-api/production-setup.rst

```diff
@@ -207,38 +198,38 @@ smaller active contract datasets, where re-initializing the cache is cheap and f
 Finally we recommend using orchestration systems or load balancers which monitor the health of
 the service and perform subsequent operations to ensure availability. These systems can use the
 `healthcheck endpoints <https://docs.daml.com/json-api/index.html#healthcheck-endpoints>`__
-provided by the *HTTP JSON API* server. This can also be tied into supporting arbitrary
-autoscaling implementation to ensure minimum number of *HTTP JSON API* servers on
+provided by the *HTTP JSON API* server. This can also be tied into supporting an arbitrary
+autoscaling implementation in order to ensure a minimum number of *HTTP JSON API* servers on
 failures.
 
 Set up the HTTP JSON API Service to work with Highly Available Participants
 ---------------------------------------------------------------------------
 
-In case the participant node itself is configured to be highly available, depending on the setup you might want
+In case the participant node itself is configured to be highly available, depending on the setup you may want
```

Comment:
I don't know that we want to change the voice. We've been referring to the user setting this up until now, rather than giving the setups agency. I think it's stylistic though so whatever you prefer.

Reply:
No harm in leaving it as is, this was another change intended to reduce length.